### ISSUE TITLE
Julia Astro tutorials

### ISSUE THREAD

**Issue by Sov-trotter** [JuliaAstro/AstroBase.jl#33]
[Didn't know where else to create this issue]
I recently came across the [astropy](http://learn.astropy.org/) tutorials website. 
Can we implement a similar thing in julia ? 
With the language getting better and stable and an exponentially increasing user base it'd be nice to have a bunch of tutorials covering the basic aspects of the package, examples, devs' guide etc.
@nolta @giordano @timholy  @kbarbary @helgee

**Comment by helgee** [comment]
Yes, "we" can certainly do that 😉 While quite necessary, it is also a tremendous amount of work and as it stands we are all volunteers.

With the caveat above in mind, I will say that I plan to do this for this specific package similarly to the already existing tutorial for [AstroTime.jl](https://juliaastro.github.io/AstroTime.jl/stable/tutorial/). AstroBase.jl is just not stable enough, yet.

**Comment by Sov-trotter** [comment]
Thanks for the prompt reply! 
I guess it's a good approach to begin with a stable package.
Do you have anything in mind in particular?
